examples/green-house-migration/legacy/greenhouse/batch/jobs.py

```
@@ -0,0 +1,48 @@
+
+
+class JobsProcessor(BaseProcessor):
+    """Processor for fetching jobs and their related data."""
+
+    entity = "jobs"
+
+    def fetch(self):
+        jobs = fetch_all_from_api("jobs")
+
+        for _job in jobs:
+            job_id = job.get("id")
```

Ruff (0.12.2)

Undefined name `job`

(F821)